crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/MaatCourtDataService.java

```diff
@@ -68,17 +68,6 @@ public Contribution updateContribution(UpdateContributionRequest request) {
         return response;
     }
 
-    public BigDecimal getContributionAppealAmount(GetContributionAmountRequest request) {
-        BigDecimal response = maatAPIClient.get(
-                new ParameterizedTypeReference<>() {
-                },
-                configuration.getMaatApi().getContributionEndpoints().getGetAppealAmountUrl(),
```

Comment:
I don't think we need this url any more so we can remove it from the configuration.

Reply:
Yeh no worries, you'll need to remove the value in the application.yaml file and also where the value is imported into the configuration in the ServicesConfiguration.java. There might be some instances of lingering around in the tests aswell so worth checking for that 👍

Reply:
Awesome thanks Nigel, done! (I wasn't sure if there were other places this endpoint exists, such as Helm config, but judging by how it's constructed in the application.yaml file I'm guessing not)